### PR TITLE
Slow Games blogpost

### PR DESCRIPTION
**Pull request by 0xapriori**
Please add any feedback for review.

### CODE REVIEW

**Comment by 0xapriori** [comment]
github rendering: https://github.com/anoma/anoma-blog/blob/apriori/slow-games/apriori/slow-games.md

File: apriori/slow-games.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
+---
+title: Slow Games
+category: distributed systems
+publish_date: 2024-08-21
+image: media/slowgamescover.png
+excerpt: This article is a summary of *Slow Games; Policy Enforcement under Uncertainty*. We explore the slow game by reviewing a conceptual framework, lossy compression model, an example game model, simulations, and future directions. 
+---
+## Preface
+This art is a summary based on the Anoma Research Topic titled *Slow Games: Policy Enforcement under Uncertainty* by D Reusche, Christopher Goes and Nicolas Della Penna. All credit to the authors. Thank you to D for feedback and review. Any errors *are* my own.
```
Comment (cwgoes) [inline apriori/slow-games.md]:
```suggestion
This article is a summary based on the Anoma Research Topic titled *Slow Games: Policy Enforcement under Uncertainty* by D Reusche, Christopher Goes and Nicolas Della Penna. All credit to the authors. Thank you to D for feedback and review. Any errors *are* my own.
```

File: apriori/slow-games.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+## Introduction
+The Slow Game is a type of principal-agent problem. The principal-agent problem ([PAP](https://en.wikipedia.org/wiki/Principal%E2%80%93agent_problem)) arises when one entity takes actions on behalf of another entity, where there is a conflict of interest.
```
Comment (cwgoes) [inline apriori/slow-games.md]:
```suggestion
The Slow Game is a type of principal-agent problem. The principal-agent problem ([PAP](https://en.wikipedia.org/wiki/Principal%E2%80%93agent_problem)) arises when one entity takes actions on behalf of another entity, where there is a potential conflict of interest. 
```

File: apriori/slow-games.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+This art is a summary based on the Anoma Research Topic titled *Slow Games: Policy Enforcement under Uncertainty* by D Reusche, Christopher Goes and Nicolas Della Penna. All credit to the authors. Thank you to D for feedback and review. Any errors *are* my own.
+
+## Introduction
+The Slow Game is a type of principal-agent problem. The principal-agent problem ([PAP](https://en.wikipedia.org/wiki/Principal%E2%80%93agent_problem)) arises when one entity takes actions on behalf of another entity, where there is a conflict of interest. 
+
+There are two actors in the slow game. 
+- Operator - operator of a service. This is the agent. 
+- Regulator - attempts to enforce policies of the service being provided by the operator. This is the principal.
+
+The regulator is slower to act than the operator, and in some cases less sophisticated. Critically, the regulator is slower in measuring the operator's actions. This introduces uncertainty depending on the difference in speed between the two actors.
```
Comment (cwgoes) [inline apriori/slow-games.md:18]: I would include some examples of Slow Games here, e.g.:
- validator set selection
- solver selection

File: apriori/slow-games.md [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+## Introduction
+The Slow Game is a type of principal-agent problem. The principal-agent problem ([PAP](https://en.wikipedia.org/wiki/Principal%E2%80%93agent_problem)) arises when one entity takes actions on behalf of another entity, where there is a conflict of interest. 
+
+There are two actors in the slow game. 
+- Operator - operator of a service. This is the agent. 
+- Regulator - attempts to enforce policies of the service being provided by the operator. This is the principal.
+
+The regulator is slower to act than the operator, and in some cases less sophisticated. Critically, the regulator is slower in measuring the operator's actions. This introduces uncertainty depending on the difference in speed between the two actors. 
+
+The Slow game report introduces a framework inspired by lossy compression problems to model this type of game. The report presents results of simulations (empirical analysis) from a minimal example. The report concludes by examining future directions of research including; composition of slow games, operator collusion in slow games, and applied modeling for _Anoma_.
```
Comment (cwgoes) [inline apriori/slow-games.md]:
```suggestion
The Slow Game report introduces a framework inspired by lossy compression problems to model this type of game. The report presents results of simulations (empirical analysis) from a minimal example. The report concludes by examining future directions of research including composition of slow games, operator collusion in slow games, and applied modeling for _Anoma_.
```

File: apriori/slow-games.md [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+- What is the policy $p$?
+- What is the deviation between the reward profile of the actions which best maximize the fast agent's returns? We refer to this as slack, but colloquially could be called MEV. 
+
+![](media/slowgame1.png)
+
+### Example: Solver Selection in Anoma
+
+In the discourse, the term *solver* is often understood as market participants who provide counterparty discovery for user intents ([Davids et al., 2023](https://perridonventures.xyz/publications/redefining-blockchain-interactions-the-crucial-role-of-solvers-in-an-Intent-focussed-future#expandingthescope-newfrontforsolvers)). Another definition could be, solvers are agents who compete to satisfy user orders which may include complicated user specified conditions ([Chitra et al., 2024](https://arxiv.org/html/2403.02525v2)). 
+
+Typically, the term solver is meant to encompass an entity that:
```
Comment (cwgoes) [inline apriori/slow-games.md]:
```suggestion
Typically, the term solver is meant to encompass an entity that performs the following actions:
```

File: apriori/slow-games.md [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
+
+>We thank the reviewers of the [Agentic Markets](https://agenticmarkets.xyz) Workshop at ICML 2024 for their helpful feedback on improving exposition and refining this work.
+
+Thank you to the teams in the industry who are exploring this topic. 
+- [Accountability along the transaction supply chain](https://ideas.skip.build/t/accountability-along-the-transaction-supply-chain/89) 
+- [dydx v4 Social Mitigation Strategy for MEV](https://dydx.forum/t/dydx-v4-social-mitigation-strategy-for-mev/1377)
+- [Cow Protocol Social Consensus Rules](https://v1.docs.cow.fi/solvers/in-depth-solver-specification/social-consensus-rules)
+- [Osmosis governance stance against off-chain harmful MEV collusion and extraction software](https://commonwealth.im/osmosis/discussion/7812-osmosis-governance-stance-against-offchain-harmful-mev-collusion-and-extraction-software)
+- [Jito Labs suspends mempool Functionality](https://blockworks.co/news/jito-labs-suspends-mempool-functionality)
+- [How I learned to stop worryng and love MEV](https://medium.com/dragonfly-research/dr-reorg-or-how-i-learned-to-stop-worrying-and-love-mev-2ee72b428d1d)
```
Comment (cwgoes) [inline apriori/slow-games.md]:
```suggestion
- [How I learned to stop worrying and love MEV](https://medium.com/dragonfly-research/dr-reorg-or-how-i-learned-to-stop-worrying-and-love-mev-2ee72b428d1d)
```

**Review by cwgoes — APPROVED** [review]
Mostly minor comments. Modulo resolution of those LGTM!

**Pull request merged**